Winium/Winium.StoreApps.Driver/CommandLineOptions.cs

```diff
@@ -33,6 +33,9 @@ internal class CommandLineOptions
         [Option("nodeconfig", Required = false, HelpText = "configuration JSON file to register driver with selenium grid")]
         public string NodeConfig { get; set; }
 
+        [Option("dependency", Required = false, HelpText = "dependencies to be installed before main app")]
+        public string Dependency { get; set; }
```

Comment:
Oh, sorry. This wasn't meant to be a part of this commit. It's how I did it at first, but then I though it would be better to put it in capabilities. I'll remove it